Solutions/GoogleCloudPlatformDNS/Data Connectors/GCP_DNS_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "GCPDNSDataConnector",
+    "title": "Google Cloud Platform DNS",
+    "publisher": "Google",
+    "descriptionMarkdown": "The Google Cloud Platform DNS data connector provides the capability to ingest [Cloud DNS query logs](https://cloud.google.com/dns/docs/monitoring#using_logging) [and Cloud DNS audit logs](https://cloud.google.com/dns/docs/audit-logging) into Azure Sentinel using the GCP Logging API. Refer to [GCP Logging API documentation](https://cloud.google.com/logging/docs/api) for more information.",
+    "additionalRequirementBanner": ">This data connector depends on a parser based on a Kusto Function to work as expected [**imDNSGCPCloudDNS**](https://aka.ms/sentinel-GCPDNSDataConnector-parser) which is deployed with the Azure Sentinel Solution.",
```

Comment:
As per the latest template there is change in additional requirement banner text could yo please update it. You can take a reference from [here](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/DataConnector_API_AzureFunctionApp_template.json)

Thanks,
Aditi Maheshwari